docs/authoring/container-skills.md

```diff
@@ -128,6 +128,12 @@ The following environment variables are available inside the container:
     Storage bucket to write objects to
 -   `ATOMIST_GRAPHQL_ENDPOINT=<url for querying the GraphQL API>` - Url to the
     Atomist GraphQL endpoint
+-   `ATOMIST_REPOSITORY_SLUG` - Slug _owner/repo name_ of repository that 
+    triggered the skill
+-   `ATOMIST_REPOSITORY` - Name of repository that triggered the skill
+-   `ATOMIST_REPOSITORY_OWNER` - Owner or organization of repository that 
+    triggered this skill
+-   `ATOMIST_REF` - SHA, branch or tag of the event that triggered the skill
```

Comment:
When would you ever get the branch? Wouldn't a push of commits or tags always be the trigger?

Reply:
We have triggers like Branch or DeletedBranch that purely trigger off of branch related events.